docs/api_requests/block-caching.md

```diff
@@ -0,0 +1,70 @@
+# Block device caching strategies
+
+Firecracker offers the possiblity of choosing the block device caching
+strategy. Caching strategy affects the path data written from inside the
+microVM takes to the host persistent storage.
+
+## How it works
+
+When installing a block device through a PUT /drives API call, users can choose
+the caching strategy by inserting a `cache_type` field in the JSON body of the
+request. The available cache types are:
+
+- `Unsafe`
+- `Writeback`
+
+### Unsafe mode (default)
+
+When configuring the block caching strategy to `Unsafe`, the device will
+advertise the VirtIO `flush` feature to the guest driver. If negotiated when
+activating the device, the guest driver will be able to send flush requests
+to the device, but the device will just acknowledge the request without
+actually performing any flushing on the host side. The data which was not
+yet committed to disk will continue to reside in the host page cache.
```

Comment:
Maybe note here that the data may is only committed to the physical disk when the host driver decides to

Reply:
I think an explanation of how caching works on Linux systems is not appropriate here, especially since whatever we write in this section highly depends on how the host OS and FS are set up. The fact that IO is cached is the default behavior in Linux systems and I don't think we need to explain this here.

We should, however, touch on this in the general block device documentation and provide our recommendations regarding host OS and FS setup. In that section we should also mention what host configuration we are running our tests on, and that should be enough.